common/src/main/java/org/opensearch/ml/common/transport/model_group/MLUpdateModelGroupInput.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.model_group;
+
+import lombok.Builder;
+import lombok.Data;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.opensearch.common.xcontent.XContentParserUtils.ensureExpectedToken;
+
+@Data
+public class MLUpdateModelGroupInput implements ToXContentObject, Writeable {
+
+    public static final String MODEL_GROUP_ID = "model_group_id";
+    public static final String NAME = "name";
+    public static final String DESCRIPTION = "description";
+    public static final String BACKEND_ROLES = "backend_roles";
+
+
+    private String modelGroupID;
```

Comment:
So user can't update tags?

Reply:
Few more fields are to be added here including tags and access fields. Will publish them in next PR